test/Microsoft.AspNetCore.Mvc.FunctionalTests/Microsoft.AspNetCore.Mvc.FunctionalTests.csproj

```diff
@@ -29,6 +28,8 @@
     <ProjectReference Include="..\..\benchmarkapps\BasicViews\BasicViews.csproj" />
     <ProjectReference Include="..\..\samples\MvcSandbox\MvcSandbox.csproj" />
 
+    <ProjectReference Include="..\Microsoft.AspNetCore.Mvc.Core.TestCommon\Microsoft.AspNetCore.Mvc.Core.TestCommon.csproj" />
```

Comment:
@javiercn @pranavkm - is there a reason why this wasn't already here?

Reply:
Mostly because it did not use any of the test types from there. Seems fine to reference it